brave/brave-core
# Implements new persona api

Resolves https://github.com/brave/brave-browser/issues/8428

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
plan 1:
- enable rewards
- make sure that wallet is created
- open Preference file and make sure that `brave.rewards.wallet` has `payment_id` and `seed` populated

plan 2:
- enable rewards
- try to restore wallet
- make sure that wallet is restored correctly

plan 3:
- enable rewards
- open restore modal
- put 16 random words in there
- make sure that you see migration message

plan 4:
- enable rewards
- restore wallet with user funds
- connect kyc uphold account
- make sure that funds are transferred 

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
